### ISSUE TITLE
OGS Links show up as "Play Go at online-go.com! | OGS" in the Forums

### ISSUE THREAD

**Issue by benjaminpjones** [online-go/online-go.com#1422]
TL;DR Please populate `og:title` with something descriptive!

**Describe the bug**
The forums try to pull relevant page data from websites that are posted to the forum.  When a link is not on its own line, the link text is changed to the [Open Graph](https://ogp.me) title.  The title for OGS is "Play Go at online-go.com! | OGS", but I believe this could be more descriptive.

Here's how all OGS links show up in the forums (when on the same line as other text):

<img width="385" alt="Screen Shot 2021-03-09 at 12 15 57 PM" src="https://user-images.githubusercontent.com/25233703/110510507-3f182800-80d1-11eb-943a-f9dfac06ffcf.png">

**To Reproduce**
1. Go to Forums
2. Make a comment with a link in it, but make sure it is on the same line as some other text.  e.g. ("Check out the Honinbo tournament: https://online-go.com/tournament/70977")
3. Note that when you post it, it shows up as "Play Go at online-go.com | OGS"

**Expected behavior**
A link like https://online-go.com/tournament/70977 should show up as "Honinbo Main Title Tournament 2021" or "Honinbo Main Title Tournament 2021 | OGS".

**Additional Context**

I looked into fixing this, but `OG_TITLE` seems to populated differently in prod (Gulpfile.js just populates it with an empty string), and I also can't tell if API calls in the Gulpfile are Kosher.

**Comment by benjaminpjones** [comment]
That's really great, thanks @anoek :)

One issue: somehow Discourse is still pulling "Play Go at online-go.com | OGS", I wonder if it is pulling the first value of <title> instead of <og:title>.  Would it be worth using the same value for `PAGE_TITLE` as well?  I know this is more of a failing on Discourse's end, but since OGS uses discourse for the forums I wanted to bring it up.

Discourse (both links to the same game):

<img width="782" alt="Screen Shot 2021-03-18 at 1 27 59 PM" src="https://user-images.githubusercontent.com/25233703/111671137-ff4af200-87ee-11eb-821a-42218abacb2f.png">

The DOM before JavaScript runs:

<img width="874" alt="Screen Shot 2021-03-18 at 1 32 35 PM" src="https://user-images.githubusercontent.com/25233703/111671166-0b36b400-87ef-11eb-88a5-a364cb38dd00.png">

**Comment by anoek** [comment]
The fix isn't live yet, i'm actually surprised it closed the issue since it was cross repository, but I guess github is cool like that.

When it goes live, it should fix this issue.

**Comment by benjaminpjones** [comment]
> The fix isn't live yet, i'm actually surprised it closed the issue since it was cross repository, but I guess github is cool like that.

Oh got it, I didn't realize `og:title` was already set for games.  Thanks again for addressing!

**Comment by anoek** [comment]
Ok it's live now, quick test showed the game name as expected now :)

**Comment by benjaminpjones** [comment]
Truly glorious!

<img width="795" alt="Screen Shot 2021-03-18 at 2 25 01 PM" src="https://user-images.githubusercontent.com/25233703/111677710-dbd77580-87f5-11eb-8387-ed431777d1dc.png">
